bower.json

```diff
@@ -2,30 +2,26 @@
   "name": "vagrant",
   "version": "0.0.0",
   "dependencies": {
-    "angular": "^1.3.0",
+    "angular": "1.4.1",
     "bootstrap-sass-official": "^3.2.0",
-    "angular-animate": "^1.3.0",
-    "angular-aria": "^1.3.0",
-    "angular-cookies": "~1.3.15",
-    "angular-messages": "^1.3.0",
-    "angular-resource": "^1.3.0",
-    "angular-route": "^1.3.0",
-    "angular-sanitize": "^1.3.0",
-    "angular-touch": "^1.3.0",
+    "angular-animate": "^1.4.1",
+    "angular-aria": "^1.4.1",
+    "angular-cookies": "^1.4.1",
+    "angular-messages": "^1.4.1",
+    "angular-resource": "^1.4.1",
+    "angular-sanitize": "^1.4.1",
+    "angular-touch": "^1.4.1",
     "angular-audio": "~1.6.2",
-    "angular-timer": "~1.3.1",
-    "angular-ui-router": "~0.2.14",
+    "angular-timer": "~1.3.3",
+    "angular-ui-router": "~0.2.15",
     "ui-router-extras": "~0.0.13"
   },
   "devDependencies": {
-    "angular-mocks": "^1.3.0",
-    "angular-gravatar": "~0.3.1",
```

Comment:
Gravatar is not used anymore as avatar provider, we use the avatars provided by soundcloud.